tests/src/e2e/ledger_tests.rs

```diff
@@ -60,6 +58,55 @@ fn run_ledger() -> Result<()> {
     Ok(())
 }
 
+#[test]
+/// In this test we:
+/// 1. Run a single genesis validator node
+/// 2. Submit a valid token transfer tx
+/// 3. Check that the transfer was processed
+fn test_simple_transfer() -> Result<()> {
```

Comment:
This test is only here to help with debugging, but it could be added (also to `main`) in a separate PR.